File: ios/Starter/Adapter/Mapper/UserMapper.swift
//
//  UserMapper.swift
//  Starter
//
//  Created by Colton Nohelty on 2/3/19.
//  Copyright © 2019 Colton Nohelty. All rights reserved.
//

import StarterCore

class UserMapper {
    func mapUserResponse(userResponse: UserResponse) -> User {
        return User(id: userResponse.id, email: userResponse.email)
    }
}

File: common/src/main/kotlin/com/noheltcj/starter/adapter/mapper/UserMapper.kt
package com.noheltcj.starter.adapter.mapper

import com.noheltcj.starter.adapter.entity.UserResponse
import com.noheltcj.starter.di.Inject
import com.noheltcj.starter.entity.User

class UserMapper @Inject constructor() {
    fun mapUserResponse(userResponse: UserResponse): User {
        return User(
            id = userResponse.id,
            email = userResponse.email
        )
    }
}